npm/ng-packs/packages/theme-shared/src/lib/components/internet-connection-status/internet-connection-status.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component } from '@angular/core';
+import { InternetConnectionService } from '../../../../../core/src/lib/services/internet-connection-service';

```

Comment:
We can use during import `@abp/ng.core` instead

Reply:
`InternetConnectionStatusComponent` also need doing the same